group-charter.html

```diff
@@ -379,17 +379,17 @@ <h3 id="milestones">Milestones</h3>
         <h2 id="coordination">Dependencies and Liaisons</h2>
         <p>HTML depends on, and is depended on by, many specifications.</p>
         <p>Many specifications in many Working Groups depend on WebIDL.</p>
-        <p>Web Sockets depends on work in the 
+        <p>Web Sockets depends on work in the
           <a href="http://www.ietf.org/dyn/wg/charter/hybi-charter.html">IETF's HyBi Working Group</a>. </p>
 	<p>This working group depends on review of aspects such as accessibility, internationalisation and privacy in its
           deliverables</p>
-          
+
         <p>The Web Platform Working Group will keep contact with and where applicable request review from at least the following:</p>
         <dl>
           <dt><a href="https://www.w3.org/WAI/APA">Accessible Platform Architectures Working Group</a></dt>
             <dd>This Group ensures W3C specifications provide support for accessibility to people with disabilities.</dd>
           <dt><a href="https://www.w3.org/WAI/ARIA">Accessible Rich Internet Applications Working Group</a></dt>
-            <dd>This Group develops ARIA</dd>
+            <dd>This group develops ARIA, a technology that is integrated into HTML and developed on a coordinated timeline, and also co-develops the HTML Accessibility API Mappings.</dd>
```

Comment:
I thought we were aiming to let our timelines be independent.

We have an issue in the text about whether we should continue to co-develop the HTML AAM, although I guess this is your vote for the answer being yes?

I'll raise an issue on it.